src/test/java/com/marklogic/mule/extension/AbstractFlowTester.java

```diff
@@ -52,6 +59,13 @@ List<DocumentData> runFlowForDocumentDataList(String flowName) {
         return toDocumentDataList((List<Message>) runFlowGetMessage(flowName).getPayload().getValue());
     }
 
+    List<DocumentData> runFlowForPagedDocumentData(String flowName) {
```

Comment:
Same thing as above - `runFlowForDocumentDataList` is now unused. I think most tests aren't going to care whether a PagingProvider is used or not. They just want a list of DocumentData instances back. So I think just change `runFlowForDocumentDataList` to do what this method does and revert the tests.